modules/rosa-accessing-your-cluster-quick.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+//
+// getting_started_rosa/creating-first-rosa-cluster.adoc
+
+
+[id="rosa-accessing-your-cluster-quick"]
```

Comment:
For modules, we include the `context` from the assembly that includes the module. So in this case:

`[id="rosa-accessing-your-cluster-quick_{context}"]`

Reply:
Done.